scripts/proxygen/deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PROXYGEN_COMMAND="$1"
+WORKSPACE="$2"
+ACCOUNT_WIDE="$3"
+
+sh.proxygen()
```

Comment:
??

Reply:
Yeah ill delete the extra proxygen bits - didnt finish it